Update node-pre-gyp to 0.11.0

Hi.

`lzma-native` does not install next to `phantomjs-prebuilt` in node **11.0.0**.

Anyway ... `lzma-native` also did not want to build from source, but upgrade `node-pre-gyp` to version `0.11.0` make it possible.

I guess update `node-pre-gyp` could solve first issue to.

`npm i phantomjs-prebuilt lzma-native` log:
```
/some/path$ npm i phantomjs-prebuilt lzma-native

> lzma-native@4.0.1 install /some/path/node_modules/lzma-native
> node-pre-gyp install --fallback-to-build && rimraf build

node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp WARN Tried to download(404): https://node-pre-gyp.addaleax.net/lzma-native/lzma_native-v4.0.1-node-v67-linux-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for lzma-native@4.0.1 and node@11.0.0 (node-v67 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Pre-built binaries not installable for lzma-native@4.0.1 and node@11.0.0 (node-v67 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error Connection closed while downloading tarball file 
gyp: Call to 'sh liblzma-config.sh "/some/path/node_modules/lzma-native/build" "/some/path/node_modules/lzma-native/deps/xz-5.2.3.tar.bz2"' returned exit status 77 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:345:16)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:12)
gyp ERR! System Linux 4.9.0-8-amd64
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64/lzma_native.node" "--module_name=lzma_native" "--module_path=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64" "--napi_version=3" "--node_abi_napi=napi"
gyp ERR! cwd /some/path/node_modules/lzma-native
gyp ERR! node -v v11.0.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64/lzma_native.node --module_name=lzma_native --module_path=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64 --napi_version=3 --node_abi_napi=napi' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/some/path/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:962:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
node-pre-gyp ERR! System Linux 4.9.0-8-amd64
node-pre-gyp ERR! command "/usr/local/bin/node" "/some/path/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /some/path/node_modules/lzma-native
node-pre-gyp ERR! node -v v11.0.0
node-pre-gyp ERR! node-pre-gyp -v v0.10.3
node-pre-gyp ERR! not ok 
Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64/lzma_native.node --module_name=lzma_native --module_path=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64 --napi_version=3 --node_abi_napi=napi' (1)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! lzma-native@4.0.1 install: `node-pre-gyp install --fallback-to-build && rimraf build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lzma-native@4.0.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/wojtek/.npm/_logs/2018-10-24T18_04_33_362Z-debug.log
/some/path$ gyp: Call to 'sh liblzma-config.sh "/some/path/node_modules/lzma-native/build" "/some/path/node_modules/lzma-native/deps/xz-5.2.3.tar.bz2"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:345:16)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:12)
gyp ERR! System Linux 4.9.0-8-amd64
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64/lzma_native.node" "--module_name=lzma_native" "--module_path=/some/path/node_modules/lzma-native/binding-v4.0.1-node-v67-linux-x64" "--napi_version=3" "--node_abi_napi=napi"
gyp ERR! cwd /some/path/node_modules/lzma-native
gyp ERR! node -v v11.0.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
```

Build from source log:
```
/some/path/lzma-native$ npm i

> phantomjs-prebuilt@2.1.16 install /some/path/lzma-native/node_modules/phantomjs-prebuilt
> node install.js

PhantomJS not found on PATH
Download already available at /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
Verified checksum of previously downloaded file
Extracting tar contents (via spawned process)
Removing /some/path/lzma-native/node_modules/phantomjs-prebuilt/lib/phantom
Copying extracted folder /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2-extract-1540404869679/phantomjs-2.1.1-linux-x86_64 -> /some/path/lzma-native/node_modules/phantomjs-prebuilt/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /some/path/lzma-native/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs

> lzma-native@4.0.1 install /some/path/lzma-native
> node-pre-gyp install --fallback-to-build && rimraf build

node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp WARN Tried to download(404): https://node-pre-gyp.addaleax.net/lzma-native/lzma_native-v4.0.1-node-v67-linux-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for lzma-native@4.0.1 and node@11.0.0 (node-v67 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Pre-built binaries not installable for lzma-native@4.0.1 and node@11.0.0 (node-v67 ABI, glibc) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error Connection closed while downloading tarball file 
gyp: Call to 'sh liblzma-config.sh "/some/path/lzma-native/build" "/some/path/lzma-native/deps/xz-5.2.3.tar.bz2"' returned exit status 77 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:345:16)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:12)
gyp ERR! System Linux 4.9.0-8-amd64
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/some/path/lzma-native/binding-v4.0.1-node-v67-linux-x64/lzma_native.node" "--module_name=lzma_native" "--module_path=/some/path/lzma-native/binding-v4.0.1-node-v67-linux-x64" "--napi_version=3" "--node_abi_napi=napi"
gyp ERR! cwd /some/path/lzma-native
gyp ERR! node -v v11.0.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/some/path/lzma-native/binding-v4.0.1-node-v67-linux-x64/lzma_native.node --module_name=lzma_native --module_path=/some/path/lzma-native/binding-v4.0.1-node-v67-linux-x64 --napi_version=3 --node_abi_napi=napi' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/some/path/lzma-native/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:962:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
node-pre-gyp ERR! System Linux 4.9.0-8-amd64
node-pre-gyp ERR! command "/usr/local/bin/node" "/some/path/lzma-native/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /some/path/lzma-native
node-pre-gyp ERR! node -v v11.0.0
node-pre-gyp ERR! node-pre-gyp -v v0.10.3
node-pre-gyp ERR! not ok 
Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/some/path/lzma-native/binding-v4.0.1-node-v67-linux-x64/lzma_native.node --module_name=lzma_native --module_path=/some/path/lzma-native/binding-v4.0.1-node-v67-linux-x64 --napi_version=3 --node_abi_napi=napi' (1)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! lzma-native@4.0.1 install: `node-pre-gyp install --fallback-to-build && rimraf build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lzma-native@4.0.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2018-10-24T18_14_33_548Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "lzma-native",
-  "version": "4.0.2",
+  "version": "4.0.1",
   "engines": {
     "node": ">=6.0.0"
   },

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "lzma-native",
-  "version": "4.0.1",
+  "version": "4.0.2",
   "engines": {
     "node": ">=6.0.0"
   },
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "nan": "^2.10.0",
-    "node-pre-gyp": "^0.10.3",
+    "node-pre-gyp": "^0.11.0",
     "readable-stream": "^2.3.5",
     "rimraf": "^2.6.1"
   },

